plugins/k8saudit/pkg/k8saudit/source.go

```diff
@@ -50,11 +56,47 @@ func (k *Plugin) Open(params string) (source.Instance, error) {
 	case "https":
 		return k.OpenWebServer(u.Host, u.Path, true)
 	case "": // by default, fallback to opening a filepath
-		file, err := os.Open(params)
+		trimmed := strings.TrimSpace(params)
+
+		if strings.HasSuffix(trimmed, ".log") {
```

Comment:
```suggestion
		fileInfo, err := os.Stat(trimmed)
		if err != nil {
			return nil, err
		}
		if !fileInfo.IsDir() {
```
The `.log` suffix is arbitrary and it can be restricting. Let's just check that the file is not a directory.